lib/common_components/custom_input_field.dart

```diff
@@ -77,6 +77,10 @@ class CustomInputField extends StatelessWidget {
           children: <Widget>[
             Expanded(
               child: TextField(
+                autocorrect: false, // textfield autocorrect off
+                enableSuggestions: false,
+                keyboardType: TextInputType
```

Comment:
Make this `keyboardType` parameter optional as this is a widget.

Reply:
@yahu1031 Does that mean I have to exclude `keyboardType` parameter and find some alternative solution

Reply:
Nope, Like passable parameter. Look at the `width`, `height` in line 68, 69, 70. If The parameter is optional(nullable) follow line 68.

Reply:
And also try fixing `remove spaces`

Reply:
Hi @yahu1031  I have done the requested changes. Shall I make a new PR?